checkov/common/sca/output.py

```diff
@@ -86,9 +86,9 @@ def _update_details_by_scan_data_format(
     details: dict[str, Any],
     vulnerability_details: dict[str, Any],
     sca_details: SCADetails | None = None,
-    scan_data_format: ScanDataFormat = ScanDataFormat.FROM_TWISTCLI
+    scan_data_format: ScanDataFormat = ScanDataFormat.TWISTCLI_FORMAT
 ) -> None:
-    if scan_data_format == ScanDataFormat.FROM_TWISTCLI:
+    if scan_data_format == ScanDataFormat.TWISTCLI_FORMAT or scan_data_format == ScanDataFormat.DEPENDENCY_TREE_FORMAT:
```

Comment:
it's cleaner to write:
if scan_data_format in (ScanDataFormat.TWISTCLI_FORMAT, ScanDataFormat.DEPENDENCY_TREE_FORMAT)